shared/Microsoft.Extensions.ActivatorUtilities.Sources/ActivatorUtilitiesConstructorAttribute.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+#if ActivatorUtilities_In_DependencyInjection
+namespace Microsoft.Extensions.DependencyInjection
+#else
+namespace Microsoft.Extensions.Internal
```

Comment:
Is this supposed to be right source for `ActivatorUtiltiies`? If so, could leave it as internal in both cases and force consumers to switch to the `.Internal` implementation? Presumably we want to eventually deprecate one of these, no?

Reply:
ActivatorUtilities is public API in DI.Abstractions and I don't think we are planning on deprecating it.

They are both required because some repos don't want to take a dependency on DI.Abstractions